recipes/sdl/all/conanfile.py

```diff
@@ -184,7 +184,7 @@ def build_requirements(self):
             # set. This could be because you are using a Mac OS X version less than 10.5
             # or because CMake's platform configuration is corrupt.
             # FIXME: Remove once CMake on macOS/M1 CI runners is upgraded.
-            self.build_requires("cmake/3.22.0")
+            self.tool_requires("cmake/3.25.2")
```

Comment:
it can be removed I think, it's an old workaround which is now useless.

Reply:
@jcar87 ?

Reply:
I'm happy to remove this if I can reproduce the original error and verify that it doesn't happen anymore - otherwise I'd rather keep it out of caution. Are there any details about what exactly was wrong?

Reply:
It's an issue related to macOS M1 when install_name of dependencies is `@rpath`, fixed in CMake 3.17 as far as I remember. MacOS CI agents have been updated with CMake 3.17 to avoid to add cmake to build requirements.

Reply:
So it seems the comment explaining the rational of this build requirement is wrong (a copy/paste?).